packages/@react-aria/autocomplete/src/useAutocomplete.ts

```diff
@@ -105,9 +106,23 @@ export function useAutocomplete<T>(props: AriaAutocompleteOptions<T>, state: Aut
     return () => clearTimeout(timeout.current);
   }, []);
 
+  useEffect(() => {
+    let handlePointerDown = (e: PointerEvent) => {
+      lastPointerType.current = e.pointerType;
+    };
+
+    if (typeof PointerEvent !== 'undefined') {
+      document.addEventListener('pointerdown', handlePointerDown, true);
```

Comment:
Realized one case this won't handle: if the last pointer event happened before the autocomplete mounted. We may need to track the pointer type globally. useFocusVisible does this already but only stores "pointer" rather than the actual pointer type...